### ISSUE TITLE
A recursive call in subscribe？

### ISSUE THREAD

**Issue by Haxxxxn** [eclipse-paho/paho.mqtt.java#983]
- [✓ ] Bug exists Release Version 1.2.5 ( Master Branch)
![image](https://user-images.githubusercontent.com/124014917/215649765-73ea1ba2-33c2-4c39-8859-b8879b8de3a3.png)
A recursive call in subscribe ?

**Comment by watta90** [comment]
Yes it's a bug in `Mqttv5` release `1.2.5` however it has been fixed in `master`, see commit  https://github.com/eclipse/paho.mqtt.java/commit/6e21aa9d667b409255d70be970b5363b1ce370a1

**Comment by inad9300** [comment]
This is great. Is there a release planned that will include this fix?
